ovs_offload/README.md

```diff
@@ -1,11 +1,30 @@
-# OVS Offload Scripts
+# Linux Networking with OVS Offload
 
 ## Introduction
 
-- The ovs_offload_lnw.py script can be used on an host server connected with Intel® Infrastructure Processing Unit via PCIe.
-- It creates the configuration to run the Linux Networking Recipe with OVS Offload on the ACC.
-- It can be used to start infrap4d on the ACC and use the script generated p4rt-ctl rules to configure ACC Port representors for the Host IDPF interfaces and IPU Physical Ports.
-- It can set up OVS bridges on the ACC using the port representors and configure VMs on the IDPF interfaces on the Host.
+- The ovs_offload_lnw.py tool can be used on an host server connected with Intel® Infrastructure Processing Unit via PCIe.
+- It is supported on IPU SDK Release version >= 1.7.0 and makes use of the P4 package fxp-net_linux-networking.pkg
+- It creates the configuration to run the Linux Networking Recipe with OVS Offloaded to the IPU ACC.
+- It can be used to start infrap4d on the ACC and program the p4rt-ctl runtime rules to configure ACC Port representors for the Host IDPF interfaces and IPU Physical Port 0 and Port 1.
+- It can set up OVS bridges on the ACC using the port representors and configure VM namespaces for the IDPF interfaces on the Host.
+- Transport (Default mode): The tool generates the OVS bridge configuration for transport Mode with IPv4 encapsulation, use helper script 6_acc_ovs_bridge.sh.
```

Comment:
```suggestion
- Transport (Default mode): The tool generates the OVS bridge configuration for transport mode with IPv4 encapsulation, use helper script 6_acc_ovs_bridge.sh.
```

Reply:
```suggestion
- Transport (Default mode): The tool generates the OVS bridge configuration for transport Mode with IPv4 encapsulation. Use helper script 6_acc_ovs_bridge.sh.
```
Comma splice. The two clauses express different thoughts and have different structures. They should be separate sentences.